src/bin/ion/commands/beta/generate/templates/rust/import.templ

```diff
@@ -0,0 +1 @@
+use ion_rs::{IonResult, IonReader, Reader, IonWriter, StreamItem};
```

Comment:
(🗺️ PR tour) This template points to the common import statement for all generated data models which has imports for Ion reader, writer etc.